lang/src/org/partiql/lang/ast/passes/AstWalker.kt

```diff
@@ -116,6 +117,7 @@ open class AstWalker(private val visitor: AstVisitor) {
                     }
                 }
                 is CreateTable, is DropTable, is DropIndex -> case { }
+                is Exec         -> throw UnsupportedOperationException("EXEC clause is not supported by the V0 AST")
```

Comment:
1.  Copy-pasta from the `V0` serializer?
1. We do need to support this in the `AstWalker` for time being, so we shouldn't be throwing an exception here.  Probably, an empty `case { }` like the previous line is appropriate.

Reply:
1. Yeah that was copied from the `AstSerialization` code.
2. Changed to `case { }`.